global_storage.lua

```diff
@@ -113,9 +113,9 @@ end
 
 -- Check if the player's global mailbox has room for an item.
 function mailbox.room_for_item(player, stack)
-	local name, player = get_player(player)
-	if player then
-		return player:get_inventory():room_for_item("mailbox", stack)
+	local name, playername = get_player(player)
```

Comment:
```suggestion
	local name
	name, player = get_player(player)
```

Reply:
Should I change the other variables that I modified to `player`?

Reply:
Yes